homework.js

```diff
@@ -2,25 +2,52 @@
 // declare a variable called fruit
 // assign your variable the value 6
 // console log your variable
+myVar = fruit {
+  let fruit = 6
+}
+console.log(fruit)
 
-
-// write a function called product that accepts two numbers and 
-// logs the product in the console and then
-// returns the answer when you multiply them both together
-// call that function on the numbers 4 & 689 to test it
+// // write a function called product that accepts two numbers and 
+// // logs the product in the console and then
+// // returns the answer when you multiply them both together
+// // call that function on the numbers 4 & 689 to test it
+function product(numberOne, numberTwo) {
+  let(numberOne * numberTwo);
+  return product;
+}
+console.log(4 * 689)
 
 // Problem 2
 // write a function called evens that takes an array of numbers, removes 
 // all the numbers that are odd and returns a new array with only the even numbers
 // remember that even numbers have a remainder of 0 when you divide them by 2
 // call your function on this array of numbers [4, 37, 18, 29, 52, 103, 22]
+// function evens(numberArray, newArray) {
+let remainder = number % 2;
+console.log("The remainder is", remainder)
+if (remainder = 0) {
+  return true;
+} else {
+  return false;
+}
+}
 
 //Problem 3
 // write a function called five that takes a word and prints it in the console
 // five times
 // call your function on the word "pineapple" to test it
+function five(word) {
+  for (let counter = 0; counter < 5; counter++)
+    console.log("pineapple")
+}
 
 //Problem 4
 // write a function called shortWords that takes an array of words and then
 // returns a new array with only the words that are 3 letters long or less
 // call your function on this array ["basin", "pit", "cheese", "bun", "river"]
+function shortWords(wordArray) {
+  for (let words of words) {
+    words.length()
```

Comment:
You'll want to do some sort of test here. Is each word's length less than 3? `words.length()` will tell you how many words are in the array. Use `word` for each individual word.